packages/devextreme/testing/tests/DevExpress.ui.widgets.dataGrid/selection.tests.js

```diff
@@ -3926,80 +3926,104 @@ QUnit.module('Selection with views', {
 
     // T550013
     QUnit.test('Click on selected selection checkbox with shift key', function(assert) {
-        const testElement = $('#container');
+        const clock = sinon.useFakeTimers();
 
-        this.options.selection.showCheckBoxesMode = 'onClick';
-        this.setup();
-        this.columnHeadersView.render(testElement);
-        this.rowsView.render(testElement);
+        try {
```

Comment:
Question.

Why do we add the ```try``` blocks in this test fixture?

Reply:
to make sure `clock.restore()` always executes, even in case of exceptions

there are lot of examples across our code base:
https://github.com/DevExpress/DevExtreme/blob/24_2/packages/devextreme/testing/tests/DevExpress.ui/collectionWidgetParts/editingTests.js#L496

whenever we need cleanup inside test body, we wrap the cleanup in `finally` and the rest in `try`